src/main/kotlin/Enums.kt

```diff
@@ -75,6 +75,14 @@ enum class HostOptions(val shortName: String, val fullName: String, val current:
         {
             hideDayPlayers = !hideDayPlayers
         }
+    ),
+    HideRolesMode(
+        "🕶 Скрывать роли",
```

Comment:
Sad that this names are not adaptive, would be a cool improvement to replace string with lambda that receives setting value and returns name
Out of scope for this PR tho